source/aks/patching-artifactory.html.md.erb

```diff
@@ -23,130 +23,129 @@ If any issues arise during the update process, troubleshooting steps are provide
-          name: artifactory-oss
-```
-
-[Example PR] (https://github.com/hmcts/cnp-flux-config/pull/29609/files)
-
-6. Run Test:
-
-• Add the below code to kustomization.yaml file for the test to run:
+  • Before updating the PTLSBOX, conduct testing to ensure no issues
 
-```command
-patchesStrategicMerge:-../../artifactory/artifactory-sbox.yaml
-```
+  • Download the repository from [GitHub] (https://github.com/hmcts/cnp-flux-config)
 
-7. Push Changes:
-
-• Raise a PR and push changes out for review and approval
+  • File directory: apps/artifactory/artifactory/…
 
-8. Update Version:
+  • Create a new file named "artifactory-sbox.yaml" with the provided code snippet
+  
+  • New file should be in the following file path "apps/artifactory/artifactory/artifactory-sbox.yaml"
 
-• For the PTLSBOX, remove "artifactory-sbox.yaml" file and its corresponding code from kustomization.yaml
+  • Update updated-version-here with latest version eg (7.84.12)
 
+  ```command
+  apiVersion: apps/v1
+  kind: StatefulSet
+  metadata:
+    name: artifactory-oss
+    namespace: artifactory
+  spec:
+    serviceName: "artifactory"
+    replicas: 2
+    selector:
+      matchLabels:
+        app: artifactory-oss
+    template:
+      metadata:
+        labels:
+          app: artifactory-oss
+      spec:
+        containers:
+          - image: docker.bintray.io/jfrog/artifactory-oss:7.84.12
+            name: artifactory-oss
+  ```
+
+  [Example PR] (https://github.com/hmcts/cnp-flux-config/pull/32185/files)
 
-• Update the version in "artifactory.yaml" to 7.77.3
+6. Run Test:
 
-```command
-      containers:
-        - image: docker.bintray.io/jfrog/artifactory-oss:7.77.3
-          name: artifactory-oss
-          env:
-```
+  • Add the below code to kustomization.yaml file for the test to run:
 
-[Example PR] (https://github.com/hmcts/cnp-flux-config/pull/29665/files)
+  • kustomization.yaml file can be found here "apps/artifactory/sbox-intsvc/base/kustomization.yaml"
 
-9. Push Changes:
+  ```command
+  patchesStrategicMerge: 
+    - ../../artifactory/artifactory-sbox.yaml
+  ```
 
-• Raise a PR and push changes out for review and approval
+7. Push Changes:
 
-• If you can go to the [Artifactory](https://artifactory.sandbox.platform.hmcts.net/ui/repos/tree/General/) site after the upgrade and checking the packages and repositories to ensure they are working.
+  • Raise a PR and push changes out for review and approval
+  
+  • Once approve merge changes and check if version has updated in ptlsbox cluster
 
-10. Release Draft and Version Bump:
+8. Check Version on ptlsbox:
+  
+  • Verifiy ptlsbox has picked up the new version
+  
+  • Run the following command and make sure pods are running with no errors
 
-• Publish the draft release in "chart-neuvector"
+  ```command
+  patchesStrategicMerge: 
+    - ../../artifactory/artifactory-sbox.yaml
+  ```
 
-• Bump the version in "flux-config" from 1.5.4 to 1.5.5
+9. Update Version after testing:
```

Comment:
What testing?

Reply:
re worded this, testing is updating PTLSBOX before updating PTL cluster